curriculum/challenges/english/25-front-end-development/review-javascript-math/6723c463e51a2d9b747d7529.md

```diff
@@ -56,7 +56,12 @@ console.log(typeof result2); // number
 
 ## Operator Precedence 
 
-- **Definition**: Operator precedence determines the order in which operations are evaluated in an expression. Operators with higher precedence are evaluated before those with lower precedence. Values inside the parenthesis will be evaluated first and multiplication/division will have higher precedence than addition/subtraction. If the operators have the same precedence, then JavaScript will use associativity. Associativity is what tells JavaScript whether to evaluate operators from left to right or right to left. For example, the exponent operator is also right to left associative:
+- **Definition**: Operator precedence determines the order in which operations are evaluated in an expression. Operators with higher precedence are evaluated before those with lower precedence.
+- **Parentheses**: Values inside parentheses are evaluated first.
+- **Higher Precedence**: Multiplication (*), division (/), and remainder (%) have higher precedence than addition (+) and subtraction (-).
+- **Same Precedence**: If operators have the same precedence, JavaScript uses associativity to determine the evaluation order:
+- **Left-to-right associativity**: Most operators, like +, -, *, and /, are evaluated from left to right.
+- **Right-to-left associativity**: Some operators, like the exponentiation operator (**), are evaluated from right to left.
 
 ```js
```

Comment:
all of these code examples can be removed here because they are being used earlier on